base/templates/layout.html

```diff
@@ -0,0 +1,35 @@
+{% load static %}
+
+<!DOCTYPE html>
+<html>
```

Comment:
This layout file is the base template for all the HTML files. They can inherit from this.